eng/scripts/Get-PkgVersion.ps1

```diff
@@ -7,26 +7,36 @@ param (
     [string] $PackageName
 )
 
-. ${PSScriptRoot}/SdkVersion-Common.ps1
+$repoRoot = Resolve-Path "$PSScriptRoot/../..";
+. ${repoRoot}\eng\common\scripts\logging.ps1
+. ${repoRoot}\eng\scripts\SdkVersion-Common.ps1
 
 $versionFileLocation = Get-VersionHppLocaiton `
     -ServiceDirectory $ServiceDirectory `
     -PackageName $PackageName
 
 if (!$versionFileLocation) {
-    $fallback = Get-Content $RepoRoot/sdk/$ServiceDirectory/$PackageName/version.txt
+    $fallbackpath = "$RepoRoot\sdk\$ServiceDirectory\$PackageName\version.txt"
```

Comment:
Why wasn't this an issue before? Are we now getting directories that don't necessarily have a version now?

Reply:
In times past we used a `version.txt` file before the language standard called for the version information be stored in `version.hpp` ... There was a time when some of the packages used `version.txt`. However, this code path should not execute anymore as all packages have migrated off of `version.txt`.

I'm tracking removal in: https://github.com/Azure/azure-sdk-for-cpp/issues/1764